
<!DOCTYPE html>


<html lang="en" data-content_root="./" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>BrainGlobe publications &#8212; BrainGlobe</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  <!--
    this give us a css class that will be invisible only if js is disabled
  -->
  <noscript>
    <style>
      .pst-js-only { display: none !important; }

    </style>
  </noscript>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=8878045cc6db502f8baf" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=8878045cc6db502f8baf" rel="stylesheet" />

    <link rel="stylesheet" type="text/css" href="_static/css/custom.css?v=1227d6e8" />
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=8f2a1f02" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- So that users can add custom icons -->
  <script src="_static/scripts/fontawesome.js?digest=8878045cc6db502f8baf"></script>
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=8878045cc6db502f8baf" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=8878045cc6db502f8baf" />

    <script src="_static/documentation_options.js?v=d45e8c67"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="_static/copybutton.js?v=f281be69"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="_static/togglebutton.js?v=4a39c7ea"></script>
    <script src="_static/design-tabs.js?v=f930bc37"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'publications';</script>
    <link rel="canonical" href="https://brainglobe.info/publications.html" />
    <link rel="icon" href="_static/brainglobe.png"/>
    <link rel="author" title="About these documents" href="about.html" />
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Talks" href="talks.html" />
    <link rel="prev" title="Contact" href="contact.html" /> 
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  <meta name="docsearch:version" content="" />   
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <dialog id="pst-search-dialog">
    
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         placeholder="Search the docs ..."
         aria-label="Search the docs ..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form>
  </dialog>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
<div class="bd-header__inner bd-page-width">
  <button class="pst-navbar-icon sidebar-toggle primary-toggle" aria-label="Site navigation">
    <span class="fa-solid fa-bars"></span>
  </button>
  
  
  <div class="col-lg-3 navbar-header-items__start">
    
      <div class="navbar-item">

  
    
  

<a class="navbar-brand logo" href="index.html">
  
  
  
  
  
    
    
      
    
    
    <img src="_static/brainglobe.png" class="logo__image only-light" alt=""/>
    <img src="_static/brainglobe.png" class="logo__image only-dark pst-js-only" alt=""/>
  
  
    <p class="title logo__title">BrainGlobe</p>
  
</a></div>
    
  </div>
  
  <div class="col-lg-9 navbar-header-items">
    
    <div class="me-auto navbar-header-items__center">
      
        <div class="navbar-item">
<nav>
  <ul class="bd-navbar-elements navbar-nav">
    
<li class="nav-item current active">
  <a class="nav-link nav-internal" href="about.html">
    About
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="tutorials/index.html">
    Tutorials
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="documentation/index.html">
    Documentation
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="community/index.html">
    Community
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="blog/index.html">
    Blog
  </a>
</li>

            <li class="nav-item dropdown">
                <button class="btn dropdown-toggle nav-item" type="button"
                data-bs-toggle="dropdown" aria-expanded="false"
                aria-controls="pst-nav-more-links">
                    More
                </button>
                <ul id="pst-nav-more-links" class="dropdown-menu">
                    
<li class=" ">
  <a class="nav-link dropdown-item nav-internal" href="projects/index.html">
    Projects
  </a>
</li>


<li class=" ">
  <a class="nav-link dropdown-item nav-internal" href="blackcap.html">
    <no title>
  </a>
</li>

                </ul>
            </li>
            
  </ul>
</nav></div>
      
    </div>
    
    
    <div class="navbar-header-items__end">
      
        <div class="navbar-item navbar-persistent--container">
          

<button class="btn search-button-field search-button__button pst-js-only" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
 <i class="fa-solid fa-magnifying-glass"></i>
 <span class="search-button__default-text">Search</span>
 <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
</button>
        </div>
      
      
        <div class="navbar-item">

<button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button pst-js-only" aria-label="Color mode" data-bs-title="Color mode"  data-bs-placement="bottom" data-bs-toggle="tooltip">
  <i class="theme-switch fa-solid fa-sun                fa-lg" data-mode="light" title="Light"></i>
  <i class="theme-switch fa-solid fa-moon               fa-lg" data-mode="dark"  title="Dark"></i>
  <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"  title="System Settings"></i>
</button></div>
      
        <div class="navbar-item"><ul class="navbar-icon-links"
    aria-label="Icon Links">
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://github.com/brainglobe" title="GitHub" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-github fa-lg" aria-hidden="true"></i>
            <span class="sr-only">GitHub</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://twitter.com/brain_globe" title="Twitter" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-twitter fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Twitter</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://mastodon.online/@brainglobe" title="Mastodon" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-mastodon fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Mastodon</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://bsky.app/profile/brainglobe.info" title="Bluesky" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-bluesky fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Bluesky</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://brainglobe.zulipchat.com" title="Zulip (Developer chat)" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-solid fa-comments fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Zulip (Developer chat)</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://forum.image.sc/tag/brainglobe" title="image.sc (Help forum)" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-solid fa-question fa-lg" aria-hidden="true"></i>
            <span class="sr-only">image.sc (Help forum)</span></a>
        </li>
</ul></div>
      
    </div>
    
  </div>
  
  
    <div class="navbar-persistent--mobile">

<button class="btn search-button-field search-button__button pst-js-only" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
 <i class="fa-solid fa-magnifying-glass"></i>
 <span class="search-button__default-text">Search</span>
 <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
</button>
    </div>
  

  
    <button class="pst-navbar-icon sidebar-toggle secondary-toggle" aria-label="On this page">
      <span class="fa-solid fa-outdent"></span>
    </button>
  
</div>

    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <dialog id="pst-primary-sidebar-modal"></dialog>
      <div id="pst-primary-sidebar" class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
      <div class="sidebar-header-items__center">
        
          
          
            <div class="navbar-item">
<nav>
  <ul class="bd-navbar-elements navbar-nav">
    
<li class="nav-item current active">
  <a class="nav-link nav-internal" href="about.html">
    About
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="tutorials/index.html">
    Tutorials
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="documentation/index.html">
    Documentation
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="community/index.html">
    Community
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="blog/index.html">
    Blog
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="projects/index.html">
    Projects
  </a>
</li>


<li class="nav-item ">
  <a class="nav-link nav-internal" href="blackcap.html">
    <no title>
  </a>
</li>

  </ul>
</nav></div>
          
        
      </div>
    
    
    
      <div class="sidebar-header-items__end">
        
          <div class="navbar-item">

<button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button pst-js-only" aria-label="Color mode" data-bs-title="Color mode"  data-bs-placement="bottom" data-bs-toggle="tooltip">
  <i class="theme-switch fa-solid fa-sun                fa-lg" data-mode="light" title="Light"></i>
  <i class="theme-switch fa-solid fa-moon               fa-lg" data-mode="dark"  title="Dark"></i>
  <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"  title="System Settings"></i>
</button></div>
        
          <div class="navbar-item"><ul class="navbar-icon-links"
    aria-label="Icon Links">
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://github.com/brainglobe" title="GitHub" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-github fa-lg" aria-hidden="true"></i>
            <span class="sr-only">GitHub</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://twitter.com/brain_globe" title="Twitter" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-twitter fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Twitter</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://mastodon.online/@brainglobe" title="Mastodon" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-mastodon fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Mastodon</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://bsky.app/profile/brainglobe.info" title="Bluesky" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-brands fa-bluesky fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Bluesky</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://brainglobe.zulipchat.com" title="Zulip (Developer chat)" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-solid fa-comments fa-lg" aria-hidden="true"></i>
            <span class="sr-only">Zulip (Developer chat)</span></a>
        </li>
        <li class="nav-item">
          
          
          
          
          
          
          
          
          <a href="https://forum.image.sc/tag/brainglobe" title="image.sc (Help forum)" class="nav-link pst-navbar-icon" rel="noopener" target="_blank" data-bs-toggle="tooltip" data-bs-placement="bottom"><i class="fa-solid fa-question fa-lg" aria-hidden="true"></i>
            <span class="sr-only">image.sc (Help forum)</span></a>
        </li>
</ul></div>
        
      </div>
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">
<nav class="bd-docs-nav bd-links"
     aria-label="Section Navigation">
  <p class="bd-links__title" role="heading" aria-level="1">Section Navigation</p>
  <div class="bd-toc-item navbar-nav"><ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="people.html">People</a></li>
<li class="toctree-l1"><a class="reference internal" href="contact.html">Contact</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">BrainGlobe publications</a></li>

<li class="toctree-l1"><a class="reference internal" href="talks.html">Talks</a></li>
<li class="toctree-l1"><a class="reference internal" href="media.html">Media</a></li>
<li class="toctree-l1"><a class="reference internal" href="funders.html">Funders</a></li>
</ul>
</div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
      <div class="sidebar-primary-item">
<div id="ethical-ad-placement"
      class="flat"
      data-ea-publisher="readthedocs"
      data-ea-type="readthedocs-sidebar"
      data-ea-manual="true">
</div></div>
  </div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        
          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item">

<nav aria-label="Breadcrumb" class="d-print-none">
  <ul class="bd-breadcrumbs">
    
    <li class="breadcrumb-item breadcrumb-home">
      <a href="index.html" class="nav-link" aria-label="Home">
        <i class="fa-solid fa-home"></i>
      </a>
    </li>
    
    <li class="breadcrumb-item"><a href="about.html" class="nav-link">About</a></li>
    
    <li class="breadcrumb-item active" aria-current="page"><span class="ellipsis">BrainGlobe publications</span></li>
  </ul>
</nav>
</div>
      
    </div>
  
  
</div>
</div>
              
              
              
                
<div id="searchbox"></div>
                <article class="bd-article">
                   <section class="tex2jax_ignore mathjax_ignore" id="brainglobe-publications">
<h1>BrainGlobe publications<a class="headerlink" href="#brainglobe-publications" title="Link to this heading">#</a></h1>
<p>Brainreg &amp; brainglobe-segmentation (formerly brainreg-segment):</p>
<blockquote>
<div><p>Tyson, A. L., Vélez-Fort, M.,  Rousseau, C. V., Cossell, L., Tsitoura, C., Lenzi, S. C., Obenhaus, H. A., Claudi, F., Branco, T., Margrie, T. W. (2022) “Accurate determination of marker location within whole-brain microscopy images” Scientific Reports, <a class="reference external" href="https://doi.org/10.1038/s41598-021-04676-9">doi.org/10.1038/s41598-021-04676-9</a></p>
</div></blockquote>
<p>3D cell detection:</p>
<blockquote>
<div><p>Tyson, A. L., Rousseau, C. V., Niedworok, C. J., Keshavarzi, S., Tsitoura, C., Cossell, L., Strom, M., Margrie, T. W. (2021) “A deep learning algorithm for 3D cell detection in whole mouse brain image datasets” <i>PLOS Computational Biology</i> 17(5): e1009074. <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1009074">doi.org/10.1371/journal.pcbi.1009074</a></p>
</div></blockquote>
<p>Brainrender:</p>
<blockquote>
<div><p>Claudi, F., Tyson, A. L., Petrucco, L., Margrie, T.W., Portugues, R.,  Branco, T. (2021) “Visualizing anatomically registered data with Brainrender” <i>eLife</i> 2021;10:e65751 <a class="reference external" href="https://doi.org/10.7554/eLife.65751">doi.org/10.7554/eLife.65751</a></p>
</div></blockquote>
<p>BrainGlobe Atlas API:</p>
<blockquote>
<div><p>Claudi, F., Petrucco, L., Tyson, A. L., Branco, T., Margrie, T. W., Portugues, R. (2020) “BrainGlobe Atlas API: a common interface for neuroanatomical atlases” <i>Journal of Open Source Software</i>, 5(54), 2668 <a class="reference external" href="https://doi.org/10.21105/joss.02668">doi.org/10.21105/joss.02668</a></p>
</div></blockquote>
</section>
<section class="tex2jax_ignore mathjax_ignore" id="publications-citing-brainglobe">
<h1>Publications citing BrainGlobe<a class="headerlink" href="#publications-citing-brainglobe" title="Link to this heading">#</a></h1>
<p><strong>This list is automatically generated from Google Scholar</strong></p>
<p>AlSubaie, R., Wee, R. W., Ritoux, A., Mishchanchuk, K., Passlack, J., Regester, D., &amp; MacAskill, A. F. (2021). Control of parallel hippocampal output pathways by amygdalar long-range inhibition. <em>Elife</em>, <em>10</em>, e74758.</p>
<p>An, X., Matho, K., Li, Y., Mohan, H., Xu, X. H., Whishaw, I. Q., Kepecs, A., &amp; Huang, Z. J. (2022). A cortical circuit for orchestrating oromanual food manipulation. <em>BioRxiv</em>, 2022–12.</p>
<p>Argunşah, A. Ö., Erdil, E., Ghani, M. U., Ramiro-Cortés, Y., Hobbiss, A. F., Karayannis, T., Çetin, M., Israely, I., &amp; Ünay, D. (2022). An interactive time series image analysis software for dendritic spines. <em>Scientific Reports</em>, <em>12</em>(1), 12405.</p>
<p>Arshadi, C., Günther, U., Eddison, M., Harrington, K. I., &amp; Ferreira, T. A. (2021). SNT: a unifying toolbox for quantification of neuronal anatomy. <em>Nature Methods</em>, <em>18</em>(4), 374–377.</p>
<p>Athey, T. L., Wright, M. A., Pavlovic, M., Chandrashekhar, V., Deisseroth, K., Miller, M. I., &amp; Vogelstein, J. T. (2023). Brainline: An open pipeline for connectivity analysis of heterogeneous whole-brain fluorescence volumes. <em>Neuroinformatics</em>, <em>21</em>(4), 637–639.</p>
<p>Bao, R., Al-Shakarji, N. M., Bunyak, F., &amp; Palaniappan, K. (2021). Dmnet: Dual-stream marker guided deep network for dense cell segmentation and lineage tracking. <em>Proceedings of the IEEE/CVF International Conference on Computer Vision</em>, 3361–3370.</p>
<p>Beyer, J., Troidl, J., Boorboor, S., Hadwiger, M., Kaufman, A., &amp; Pfister, H. (2022). A survey of visualization and analysis in high-resolution connectomics. <em>Computer Graphics Forum</em>, <em>41</em>(3), 573–607.</p>
<p>Bhatia, H. S., Brunner, A.-D., Öztürk, F., Kapoor, S., Rong, Z., Mai, H., Thielert, M., Ali, M., Al-Maskari, R., Paetzold, J. C., &amp; others. (2022). Spatial proteomics in three-dimensional intact specimens. <em>Cell</em>, <em>185</em>(26), 5040–5058.</p>
<p>(BICCN), B. I. C. C. N., Adkins, R. S., Aldridge, A. I., Allen, S., Ament, S. A., An, X., Armand, E., Ascoli, G. A., Bakken, T. E., Bandrowski, A., &amp; others. (2020). A multimodal cell census and atlas of the mammalian primary motor cortex. <em>Biorxiv</em>, 2020–10.</p>
<p>Bimbard, C, Takács, F., Fabre, J., Melin, M., O’Neill, N., Robacha, M., Street, J., van Beest, E., Churchland, A., Harris, K., &amp; others. (2023). Reusable, flexible, and lightweight chronic implants for Neuropixels probes. <em>BioRxiv</em>.</p>
<p>Bimbard, Célian, Sit, T. P., Lebedeva, A., Reddy, C. B., Harris, K. D., &amp; Carandini, M. (2023). Behavioral origin of sound-evoked activity in mouse visual cortex. <em>Nature Neuroscience</em>, <em>26</em>(2), 251–258.</p>
<p>Bimbard, Célian, Takács, F., Catarino, J., Fabre, J., Gupta, S., Lenzi, S., Melin, M., O’Neill, N., Robacha, M., Street, J., &amp; others. (2023). An adaptable, reusable, and light implant for chronic Neuropixels probes. <em>BioRxiv</em>.</p>
<p>Birman, D., Yang, K. J., West, S. J., Karsh, B., Browning, Y., Laboratory, I. B., Siegle, J. H., &amp; Steinmetz, N. A. (2023). Pinpoint: trajectory planning for multi-probe electrophysiology and injections in an interactive web-based 3D environment. <em>BioRxiv</em>.</p>
<p>Blackmore, M., Batsel, E., &amp; Tsoulfas, P. (2021). Widening spinal injury research to consider all supraspinal cell types: Why we must and how we can. <em>Experimental Neurology</em>, <em>346</em>, 113862.</p>
<p>Blixhavn, C. H., Reiten, I., Kleven, H., Øvsthus, M., Yates, S. C., Schlegel, U., Puchades, M. A., Schmid, O., Bjaalie, J. G., Bjerke, I. E., &amp; others. (2024). The Locare workflow: representing neuroscience data locations as geometric objects in 3D brain atlases. <em>Frontiers in Neuroinformatics</em>, <em>18</em>, 1284107.</p>
<p>Bonapersona, V, Schuler, H., Damsteegt, R., Adolfs, Y., Pasterkamp, R., van den Heuvel, M., Joëls, M., &amp; Sarabdjitsingh, R. (2021). The mouse brain after foot-shock in 4D: temporal dynamics at a single-cell resolution. <em>BioRxiv</em>, 2021–05.</p>
<p>Bonapersona, Valeria, Schuler, H., Damsteegt, R., Adolfs, Y., Pasterkamp, R. J., van den Heuvel, M. P., Joëls, M., &amp; Sarabdjitsingh, R. A. (2022). The mouse brain after foot shock in four dimensions: Temporal dynamics at a single-cell resolution. <em>Proceedings of the National Academy of Sciences</em>, <em>119</em>(8), e2114002119.</p>
<p>Brown, A. P., Cossell, L., Strom, M., Tyson, A. L., Vélez-Fort, M., &amp; Margrie, T. W. (2021). Analysis of segmentation ontology reveals the similarities and differences in connectivity onto L2/3 neurons in mouse V1. <em>Scientific Reports</em>, <em>11</em>(1), 4983.</p>
<p>Buck, S. A., Rubin, S. A., Kunkhyen, T., Treiber, C. D., Xue, X., Fenno, L. E., Mabry, S. J., Sundar, V. R., Yang, Z., Shah, D., &amp; others. (2023). Sexually dimorphic mechanisms of VGLUT-mediated protection from dopaminergic neurodegeneration. <em>BioRxiv</em>.</p>
<p>Campagner, D., Vale, R., Tan, Y. L., Iordanidou, P., Pavón Arocas, O., Claudi, F., Stempel, A. V., Keshavarzi, S., Petersen, R. S., Margrie, T. W., &amp; others. (2020). A cortico-collicular circuit for accurate orientation to shelter during escape. <em>BioRxiv</em>, 2020–05.</p>
<p>Campagner, D., Vale, R., Tan, Y. L., Iordanidou, P., Pavón Arocas, O., Claudi, F., Stempel, A. V., Keshavarzi, S., Petersen, R. S., Margrie, T. W., &amp; others. (2023). A cortico-collicular circuit for orienting to shelter during escape. <em>Nature</em>, <em>613</em>(7942), 111–119.</p>
<p>Chalatsi, T., Fernandez, L. M., Scholler, J., Batti, L., Kolaxi, A., Restivo, L., Lüthi, A., Mameli, M., &amp; Nikoletopoulou, V. (2022). Autophagy in parvalbumin interneurons is required for inhibitory transmission and memory via regulation of synaptic proteostasis. <em>BioRxiv</em>, 2022–10.</p>
<p>Chen, R., Liu, M., Chen, W., Wang, Y., &amp; Meijering, E. (2023). Deep learning in mesoscale brain microscopy image analysis: A review. <em>Computers in Biology and Medicine</em>, 107617.</p>
<p>Chini, M., Hnida, M., Kostka, J. K., Chen, Y.-N., &amp; Hanganu-Opatz, I. L. (2023). Extreme distributions in the preconfigured developing brain. <em>BioRxiv</em>, 2023–11.</p>
<p>Chini, M., Pfeffer, T., &amp; Hanganu-Opatz, I. (2022). An increase of inhibition drives the developmental decorrelation of neural activity. <em>Elife</em>, <em>11</em>, e78811.</p>
<p>Chini, M., Pfeffer, T., &amp; Hanganu-Opatz, I. L. (2021). Developmental increase of inhibition drives decorrelation of neural activity. <em>BioRxiv</em>, 2021–07.</p>
<p>Chiu, C.-L., Clack, N., &amp; others. (2022). Napari: A Python multi-dimensional image viewer platform for the research community. <em>Microscopy and Microanalysis</em>, <em>28</em>(S1), 1576–1577.</p>
<p>Claar, L. D., Rembado, I., Kuyat, J. R., Russo, S., Marks, L. C., Olsen, S. R., &amp; Koch, C. (2023). Cortico-thalamo-cortical interactions modulate electrically evoked EEG responses in mice. <em>ELife</em>, <em>12</em>, RP84630.</p>
<p>Claudi, F., Petrucco, L., Tyson, A., Branco, T., Margrie, T., &amp; Portugues, R. (2020). BrainGlobe Atlas API: a common interface for neuroanatomical atlases. <em>Journal of Open Source Software</em>, <em>5</em>(54), 2668–2668.</p>
<p>Claudi, F., Tyson, A. L., &amp; Branco, T. (2020). Brainrender. <em>A Python Based Software for Visualisation of Neuroanatomical and Morphological Data. BioRxiv</em>, <em>23</em>.</p>
<p>Claudi, F., Tyson, A. L., Petrucco, L., Margrie, T. W., Portugues, R., &amp; Branco, T. (2021). Visualizing anatomically registered data with brainrender. <em>Elife</em>, <em>10</em>, e65751.</p>
<p>Cramer, T., Gill, R., Thirouin, Z. S., Vaas, M., Sampath, S., Martineau, F., Noya, S. B., Panzanelli, P., Sudharshan, T. J., Colameo, D., &amp; others. (2022). Cross-talk between GABAergic postsynapse and microglia regulate synapse loss after brain ischemia. <em>Science Advances</em>, <em>8</em>(9), eabj0112.</p>
<p>Culp, A. (2023). <em>Cell Type Diversity in the Norepinephrine Signaling Network</em> [Phdthesis]. Oregon Health.</p>
<p>Datta, M. S., Chen, Y., Chauhan, S., Zhang, J., De La Cruz, E. D., Gong, C., &amp; Tomer, R. (2023). Whole-brain mapping reveals the divergent impact of ketamine on the dopamine system. <em>Cell Reports</em>, <em>42</em>(12).</p>
<p>Davoudian, P. A., Shao, L.-X., &amp; Kwan, A. C. (2023). Shared and distinct brain regions targeted for immediate early gene expression by ketamine and psilocybin. <em>ACS Chemical Neuroscience</em>, <em>14</em>(3), 468–480.</p>
<p>De Filippo, R., &amp; Schmitz, D. (2023). Differential ripple propagation along the hippocampal longitudinal axis. <em>Elife</em>, <em>12</em>, e85488.</p>
<p>Dehaqani, A. A., Michelon, F., Patella, P., Petrucco, L., Piasini, E., &amp; Iurilli, G. (2024). A mechanosensory feedback that uncouples external and self-generated sensory responses in the olfactory cortex. <em>Cell Reports</em>, <em>43</em>(4).</p>
<p>Dehaqani, A. A., Michelon, F., Petrucco, L., Patella, P., Piasini, E., &amp; Iurilli, G. (2023). A mixed mechano-olfactory code for sniff-invariant odor representations. <em>BioRxiv</em>, 2023–04.</p>
<p>Durand, S., Heller, G. R., Ramirez, T. K., Luviano, J. A., Williford, A., Sullivan, D. T., Cahoon, A. J., Farrell, C., Groblewski, P. A., Bennett, C., &amp; others. (2023). Acute head-fixed recordings in awake mice with multiple Neuropixels probes. <em>Nature Protocols</em>, <em>18</em>(2), 424–457.</p>
<p>Englund, M., James, S. S., Bottom, R., Huffman, K. J., Wilson, S. P., &amp; Krubitzer, L. A. (2021). Comparing cortex-wide gene expression patterns between species in a common reference frame. <em>BioRxiv</em>, 2021–07.</p>
<p>Esmaeili, V., Oryshchuk, A., Asri, R., Tamura, K., Foustoukos, G., Liu, Y., Guiet, R., Crochet, S., &amp; Petersen, C. C. (2022). Learning-related congruent and incongruent changes of excitation and inhibition in distinct cortical areas. <em>PLoS Biology</em>, <em>20</em>(5), e3001667.</p>
<p>Fanton, S., &amp; Thompson, W. H. (2023). NetPlotBrain: A Python package for visualizing networks and brains. <em>Network Neuroscience</em>, <em>7</em>(2), 461–477.</p>
<p>Farrell, J. S., Hwaun, E., Dudok, B., &amp; Soltesz, I. (2024). Neural and behavioural state switching during hippocampal dentate spikes. <em>Nature</em>, 1–6.</p>
<p>Fernholz, M. (2023). <em>A search for functional connectivity rules in the visual thalamus and hippocampus</em> [Phdthesis]. lmu.</p>
<p>Fiederling, F., Hammond, L. A., Ng, D., Mason, C., &amp; Dodd, J. (2021). Tools for efficient analysis of neurons in a 3D reference atlas of whole mouse spinal cord. <em>Cell Reports Methods</em>, <em>1</em>(5).</p>
<p>Fisher, J., Verhagen, M., Long, Z., Moissidis, M., Yan, Y., He, C., Wang, J., Micoli, E., Alastruey, C. M., Moors, R., &amp; others. (2024). Cortical somatostatin long-range projection neurons and interneurons exhibit divergent developmental trajectories. <em>Neuron</em>, <em>112</em>(4), 558–573.</p>
<p>Franceschini, A., Mazzamuto, G., Checcucci, C., Chicchi, L., Fanelli, D., Costantini, I., Passani, M. B., Silva, B. A., Pavone, F. S., &amp; Silvestri, L. (2023). Brain-wide neuron quantification toolkit reveals strong sexual dimorphism in the evolution of fear memory. <em>Cell Reports</em>, <em>42</em>(8).</p>
<p>Frankowski, J. C., Tierno, A., Pavani, S., Cao, Q., Lyon, D. C., &amp; Hunt, R. F. (2022). Brain-wide reconstruction of inhibitory circuits after traumatic brain injury. <em>Nature Communications</em>, <em>13</em>(1), 3417.</p>
<p>Fuglstad, J. G., Saldanha, P., Paglia, J., &amp; Whitlock, J. R. (2023). Histological E-data registration in rodent brain spaces. <em>Elife</em>, <em>12</em>, e83496.</p>
<p>Galloni, A. R., Ye, Z., &amp; Rancz, E. (2022). Dendritic domain-specific sampling of long-range axons shapes feedforward and feedback connectivity of L5 neurons. <em>Journal of Neuroscience</em>, <em>42</em>(16), 3394–3405.</p>
<p>Gerlei, K. Z., Brown, C. M., Sürmeli, G., &amp; Nolan, M. F. (2021). Deep entorhinal cortex: from circuit organization to spatial cognition and memory. <em>Trends in Neurosciences</em>, <em>44</em>(11), 876–887.</p>
<p>Greenstreet, F., Vergara, H. M., Pati, S., Schwarz, L., Wisdom, M., Marbach, F., Johansson, Y., Rollik, L., Moskovitz, T., Clopath, C., &amp; others. (2022). Action prediction error: a value-free dopaminergic teaching signal that drives stable learning. <em>BiorXiv</em>, 2022–09.</p>
<p>Gruver, K. (2023). <em>Purkinje cells converge non-randomly on neurons in the cerebellar nuclei</em>.</p>
<p>Gruver, K. M., Jiao, J. W., Fields, E., Song, S., Sjöström, P. J., &amp; Watt, A. J. (2023). Structured connectivity in the output of the cerebellar cortex. <em>BioRxiv</em>, 2023–03.</p>
<p>Guo, S., Xue, J., Liu, J., Ye, X., Guo, Y., Liu, D., Zhao, X., Xiong, F., Han, X., &amp; Peng, H. (2022). Smart imaging to empower brain-wide neuroscience at single-cell levels. <em>Brain Informatics</em>, <em>9</em>(1), 10.</p>
<p>Hagihara, K. M., Bukalo, O., Zeller, M., Aksoy-Aksel, A., Karalis, N., Limoges, A., Rigg, T., Campbell, T., Mendez, A., Weinholtz, C., &amp; others. (2021). Intercalated amygdala clusters orchestrate a switch in fear state. <em>Nature</em>, <em>594</em>(7863), 403–407.</p>
<p>Hahn, J. D., &amp; Duckworth, C. (2023). A brain flatmap data visualization tool for mouse, rat, and human. <em>Journal of Comparative Neurology</em>, <em>531</em>(10), 1008–1016.</p>
<p>Jin, M., Nguyen, J. D., Weber, S. J., Mejias-Aponte, C. A., Madangopal, R., &amp; Golden, S. A. (2022). Smart: An open-source extension of wholebrain for intact mouse brain registration and segmentation. <em>Eneuro</em>, <em>9</em>(3).</p>
<p>Jones, A. (2023). <em>Probabilistic models for structured biomedical data</em> [Phdthesis]. Princeton University.</p>
<p>Jones, A., Cai, D., Li, D., &amp; Engelhardt, B. E. (2023). Optimizing the design of spatial genomic studies. <em>BioRxiv</em>.</p>
<p>Kaltenecker, D., Al-Maskari, R., Negwer, M., Hoeher, L., Kofler, F., Zhao, S., Todorov, M., Rong, Z., Paetzold, J. C., Wiestler, B., &amp; others. (2024). Virtual reality-empowered deep-learning analysis of brain cells. <em>Nature Methods</em>, 1–10.</p>
<p>Kantor, B., &amp; Ben-Ami Bartal, I. (2023). Brainways: An AI-based Tool for Automated Registration, Quantification and Generation of Brain-wide Activity Networks Based on Fluorescence in Coronal Slices. <em>BioRxiv</em>, 2023–05.</p>
<p>Keshavarzi, S., Bracey, E. F., Faville, R. A., Campagner, D., Tyson, A. L., Lenzi, S. C., Branco, T., &amp; Margrie, T. W. (2021). The retrosplenial cortex combines internal and external cues to encode head velocity during navigation. <em>BioRxiv</em>, 2021–01.</p>
<p>Keshavarzi, S., Bracey, E. F., Faville, R. A., Campagner, D., Tyson, A. L., Lenzi, S. C., Branco, T., &amp; Margrie, T. W. (2022). Multisensory coding of angular head velocity in the retrosplenial cortex. <em>Neuron</em>, <em>110</em>(3), 532–543.</p>
<p>Kintscher, M., Kochubey, O., &amp; Schneggenburger, R. (2023). A striatal circuit balances learned fear in the presence and absence of sensory cues. <em>Elife</em>, <em>12</em>, e75703.</p>
<p>Kostka, Johanna K, &amp; Hanganu-Opatz, I. L. (2023). Olfactory-driven beta band entrainment of limbic circuitry during neonatal development. <em>The Journal of Physiology</em>, <em>601</em>(16), 3605–3630.</p>
<p>Kostka, Johanna Katharina. (2022). <em>Olfactory entrainment of cortical-hippocampal networks during development</em> [Phdthesis]. Staats-und Universitätsbibliothek Hamburg Carl von Ossietzky.</p>
<p>Lauridsen, K., Ly, A., Prévost, E. D., McNulty, C., McGovern, D. J., Tay, J. W., Dragavon, J., &amp; Root, D. H. (2022). A semi-automated workflow for brain Slice Histology Alignment, Registration, and Cell Quantification (SHARCQ). <em>Eneuro</em>, <em>9</em>(2).</p>
<p>Lenzi, S. (2021). <em>A Role for the Tail of the Striatum in the Adaptive Control of Innate Escape Behaviour</em> [Phdthesis]. UCL (University College London).</p>
<p>Li, D. (2022). <em>Bistable Neural Dynamics Underlying Cognition and Spontaneous Activity: Computational Modeling and Empirical Analysis</em> [Phdthesis]. Yale University.</p>
<p>Lowet, A. S., Zheng, Q., Meng, M., Matias, S., Drugowitsch, J., &amp; Uchida, N. (2023). Distributional Reinforcement Learning in the Mammalian Brain. <em>UniReps: The First Workshop on Unifying Representations in Neural Models</em>.</p>
<p>Lowet, A. S., Zheng, Q., Meng, M., Matias, S., Drugowitsch, J., &amp; Uchida, N. (2024). An opponent striatal circuit for distributional reinforcement learning. <em>BioRxiv</em>, 2024–01.</p>
<p>Lu, T., Shinozaki, M., Nagoshi, N., Nakamura, M., &amp; Okano, H. (2022). 3D imaging of supraspinal inputs to the thoracic and lumbar spinal cord mapped by retrograde tracing and light-sheet microscopy. <em>Journal of Neurochemistry</em>, <em>162</em>(4), 352–370.</p>
<p>Lupori, L., &amp; others. (2023). <em>Genetic and environmental factors regulating structure, function and plasticity of the visual cortex</em>.</p>
<p>Lupori, L., Totaro, V., Cornuti, S., Ciampi, L., Carrara, F., Grilli, E., Viglione, A., Tozzi, F., Putignano, E., Mazziotti, R., &amp; others. (2023). A comprehensive atlas of perineuronal net distribution and colocalization with parvalbumin in the adult mouse brain. <em>Cell Reports</em>, <em>42</em>(7).</p>
<p>MacDowell, C. J., Libby, A., Jahn, C. I., Tafazoli, S., &amp; Buschman, T. J. (2023). Multiplexed subspaces route neural activity across brain-wide networks. <em>BioRxiv</em>.</p>
<p>manuscript editors, P., coordination, A., data analysis Armand Ethan 42 Yao Zizhen 5, I., seq data ATAC-generation, processing Fang Rongxin 45 Hou Xiaomeng 10 Lucero Jacinta D. 18 Osteen Julia K. 18 Pinto-Duarte Antonio 18 Poirion Olivier 10 Preissl Sebastian 10 Wang Xinxin 10 97 97, retro-seq data Epi-generation, processing Dominguez Bertha 53 Ito-Cole Tony 1 Jacobs Matthew 1 Jin Xin 54 99 100 99 100 Lee Cheng-Ta 53 Lee Kuo-Fen 53 Miyazaki Paula Assakura 1 Pang Yan 1 Rashid Mohammad 1 Smith Jared B. 54 Vu Minh 1 Williams Elora 54, &amp; others. (2021). A multimodal cell census and atlas of the mammalian primary motor cortex. <em>Nature</em>, <em>598</em>(7879), 86–102.</p>
<p>McGovern, D. J., Polter, A. M., Prévost, E. D., Ly, A., McNulty, C. J., Rubinstein, B., &amp; Root, D. H. (2023). Ventral tegmental area glutamate neurons establish a mu-opioid receptor gated circuit to mesolimbic dopamine neurons and regulate opioid-seeking behavior. <em>Neuropsychopharmacology</em>, <em>48</em>(13), 1889–1900.</p>
<p>McKenna, K., Prasad, S., Cooper, J., King, A. M., Shahzeidi, S., Mittal, J., Zalta, M., Mittal, R., &amp; Eshraghi, A. A. (2024). Incidence of Otolaryngological Manifestations in Individuals with Autism Spectrum Disorder: A Special Focus on Auditory Disorders. <em>Audiology Research</em>, <em>14</em>(1), 35–61.</p>
<p>Mitrevica, Z., &amp; Murray, A. J. (2023). Speed-independent modulation of locomotor gait preference by sensory feedback in mice. <em>BioRxiv</em>, 2023–03.</p>
<p>Mohan, H., An, X., Xu, X. H., Kondo, H., Zhao, S., Matho, K. S., Wang, B.-S., Musall, S., Mitra, P., &amp; Huang, Z. J. (2023). Cortical glutamatergic projection neuron types contribute to distinct functional subnetworks. <em>Nature Neuroscience</em>, <em>26</em>(3), 481–494.</p>
<p>Morimoto, M. M., Uchishiba, E., &amp; Saleem, A. B. (2021). Organization of feedback projections to mouse primary visual cortex. <em>IScience</em>, <em>24</em>(5).</p>
<p>Muñoz-Castañeda, R., Zingg, B., Matho, K. S., Chen, X., Wang, Q., Foster, N. N., Li, A., Narasimhan, A., Hirokawa, K. E., Huo, B., &amp; others. (2021). Cellular anatomy of the mouse primary motor cortex. <em>Nature</em>, <em>598</em>(7879), 159–166.</p>
<p>Negwer, M., Bosch, B., Bormann, M., Hesen, R., Lütje, L., Aarts, L., Rossing, C., Nadif Kasri, N., &amp; Schubert, D. (2023). FriendlyClearMap: an optimized toolkit for mouse brain mapping and analysis. <em>GigaScience</em>, <em>12</em>, giad035.</p>
<p>Newmaster, K. T., Kronman, F. A., Wu, Y., &amp; Kim, Y. (2022). Seeing the forest and its trees together: implementing 3D light microscopy pipelines for cell type mapping in the mouse brain. <em>Frontiers in Neuroanatomy</em>, <em>15</em>, 787601.</p>
<p>Nyberg, N., Duvelle, É., Barry, C., &amp; Spiers, H. J. (2022). Spatial goal coding in the hippocampal formation. <em>Neuron</em>, <em>110</em>(3), 394–422.</p>
<p>Paglia, J., Saldanha, P., Fuglstad, J. G., &amp; Whitlock, J. R. (2021). TRACER: a toolkit to register and visualize anatomical coordinates in the rat brain. <em>BioRxiv</em>, 2021–10.</p>
<p>Palchaudhuri, S., Osypenko, D., Kochubey, O., &amp; Schneggenburger, R. (2023). An insular cortex to lateral amygdala pathway in fear learning. <em>BioRxiv</em>, 2023–01.</p>
<p>Parks, D. F., Schneider, A. M., Xu, Y., Brunwasser, S. J., Funderburk, S., Thurber, D., Blanche, T., Dyer, E. L., Haussler, D., &amp; Hengen, K. B. (2023). A non-oscillatory, millisecond-scale embedding of brain state provides insight into behavior. <em>BioRxiv</em>.</p>
<p>Patel, V. (2021). <em>Deep Learning model to Automate the process of mapping Cancer Cells to Cell Lines &amp; Cancer Types from Single Cell RNA-Seq Data</em>.</p>
<p>Pavón Arocas, O. (2022). <em>Biophysical properties and gene expression profile of single periaqueductal gray neurons</em> [Phdthesis]. UCL (University College London).</p>
<p>Perens, J. (2021). <em>Enabling multimodal whole-brain investigation for drug discovery</em>.</p>
<p>Perens, J., &amp; Hecksher-Sørensen, J. (2022). Digital brain maps and virtual neuroscience: An emerging role for light-sheet fluorescence microscopy in drug development. <em>Frontiers in Neuroscience</em>, <em>16</em>, 866884.</p>
<p>Petrucco, L., Lavian, H., Wu, Y. K., Svara, F., Štih, V., &amp; Portugues, R. (2023). Neural dynamics and architecture of the heading direction circuit in zebrafish. <em>Nature Neuroscience</em>, <em>26</em>(5), 765–773.</p>
<p>Pochinok, I., Stöber, T. M., Triesch, J., Chini, M., &amp; Hanganu-Opatz, I. L. (2024). A developmental increase of inhibition promotes the emergence of hippocampal ripples. <em>Nature Communications</em>, <em>15</em>(1), 738.</p>
<p>Pouchelon, G., Dwivedi, D., Bollmann, Y., Agba, C. K., Xu, Q., Mirow, A. M., Kim, S., Qiu, Y., Sevier, E., Ritola, K. D., &amp; others. (2021). The organization and development of cortical interneuron presynaptic circuits are area specific. <em>Cell Reports</em>, <em>37</em>(6).</p>
<p>Prévost, E. D., Phillips, A., Lauridsen, K., Enserro, G., Rubinstein, B., Alas, D., McGovern, D. J., Ly, A., Banks, M., McNulty, C., &amp; others. (2023). Monosynaptic inputs to ventral tegmental area glutamate and GABA co-transmitting neurons. <em>BioRxiv</em>.</p>
<p>Qian, K., Friedman, B., Takatoh, J., Wang, F., Kleinfeld, D., &amp; Freund, Y. (2023). CellBoost: A pipeline for machine assisted annotation in neuroanatomy. <em>BioRxiv</em>, 2023–09.</p>
<p>Regev, A. (2021). <em>A multimodal cell census and atlas of the mammalian primary motor cortex</em>.</p>
<p>Ruff, K. M., Choi, Y. H., Cox, D., Ormsby, A. R., Myung, Y., Ascher, D. B., Radford, S. E., Pappu, R. V., &amp; Hatters, D. M. (2022). Sequence grammar underlying the unfolding and phase separation of globular proteins. <em>Molecular Cell</em>, <em>82</em>(17), 3193–3208.</p>
<p>Sanchez-Arias, J. C., Carrier, M., Frederiksen, S. D., Shevtsova, O., McKee, C., van der Slagt, E., Gonçalves de Andrade, E., Nguyen, H. L., Young, P. A., Tremblay, M., &amp; others. (2021). A Systematic, Open-Science Framework for Quantification of Cell-Types in Mouse Brain Sections Using Fluorescence Microscopy. <em>Frontiers in Neuroanatomy</em>, <em>15</em>, 722443.</p>
<p>Scholler, J., Jonsson, J., Jordá-Siquier, T., Gantar, I., Batti, L., Cheeseman, B. L., Pagès, S., Sbalzarini, I. F., &amp; Lamy, C. M. (2023). Efficient image analysis for large-scale next generation histopathology using pAPRica. <em>BioRxiv</em>, 2023–01.</p>
<p>Schweihoff, J. F. (2022). <em>Ai-based, behavior dependent approaches for connectomic reconstruction of neuronal circuits</em> [Phdthesis]. Universitäts-und Landesbibliothek Bonn.</p>
<p>Siu, P. H., Müller, E., Zerbi, V., Aquino, K., &amp; Fulcher, B. D. (2022). Extracting dynamical understanding from neural-mass models of mouse cortex. <em>Frontiers in Computational Neuroscience</em>, <em>16</em>, 847336.</p>
<p>Soltwedel, J., Suckert, T., Beyreuther, E., Schneider, M., Boucsein, M., Bodenstein, E., Nexhipi, S., Stolz-Kieslich, L., Krause, M., von Neubeck, C., &amp; others. (2023). Slice2Volume: Fusion of multimodal medical imaging and light microscopy data of irradiation-injured brain tissue in 3D. <em>Radiotherapy and Oncology</em>, <em>182</em>, 109591.</p>
<p>Szabo, G. G., Farrell, J. S., Dudok, B., Hou, W.-H., Ortiz, A. L., Varga, C., Moolchand, P., Gulsever, C. I., Gschwind, T., Dimidschstein, J., &amp; others. (2022). Ripple-selective GABAergic projection cells in the hippocampus. <em>Neuron</em>, <em>110</em>(12), 1959–1977.</p>
<p>Szelenyi, E. R., Navarrete, J. S., Murry, A. D., Zhang, Y., Girven, K. S., Kuo, L., Cline, M. M., Bernstein, M. X., Burdyniuk, M., Bowler, B., &amp; others. (2023). An arginine-rich nuclear localization signal (ArgiNLS) strategy for streamlined image segmentation of single-cells. <em>BioRxiv</em>.</p>
<p>Takatoh, J., Park, J. H., Lu, J., Li, S., Thompson, P., Han, B.-X., Zhao, S., Kleinfeld, D., Friedman, B., &amp; Wang, F. (2021). Constructing an adult orofacial premotor atlas in Allen mouse CCF. <em>Elife</em>, <em>10</em>.</p>
<p>Takatoh, J., Prevosto, V., Thompson, P., Lu, J., Chung, L., Harrahill, A., Li, S., Zhao, S., He, Z., Golomb, D., &amp; others. (2022). The whisking oscillator circuit. <em>Nature</em>, <em>609</em>(7927), 560–568.</p>
<p>Tantirigama, M. L., Schoenherr, A., Moreno-Velasquez, L., Faiss, L., Rost, B. R., Larkum, M. E., Judkewitz, B., Stumpenhorst, K., Rivalan, M., Winter, Y., &amp; others. (2023). Binge Feeding Promotes Appetite via Modulating Olfactory Flavor Representation. <em>BioRxiv</em>, 2023–10.</p>
<p>Taylor, D. (2023). <em>A brainstem circuit mechanism for coordinating the head and eyes during gaze movements</em> [Phdthesis]. University of California, San Francisco.</p>
<p>Terstege, D. J., &amp; Epp, J. R. (2022). Network neuroscience untethered: brain-wide immediate early gene expression for the analysis of functional connectivity in freely behaving animals. <em>Biology</em>, <em>12</em>(1), 34.</p>
<p>Terstege, D. J., Oboh, D. O., &amp; Epp, J. R. (2022). FASTMAP: Open-source flexible atlas segmentation tool for multi-area processing of biological images. <em>ENeuro</em>, <em>9</em>(2).</p>
<p>Thompson, E. (2024). <em>Subcortical circuits for procedural learning and memory</em> [Phdthesis]. UCL (University College London).</p>
<p>Tillmann, J. F. (n.d.). <em>Ai-based, behavior dependent approaches for connectomic reconstruction of neuronal circuits</em> [Phdthesis]. Dissertation, Bonn, Rheinische Friedrich-Wilhelms-Universität, 2022.</p>
<p>Toubal, I. E., Al-Shakarji, N., Cornelison, D., &amp; Palaniappan, K. (2023). Ensemble deep learning object detection fusion for cell tracking, mitosis, and lineage. <em>IEEE Open Journal of Engineering in Medicine and Biology</em>.</p>
<p>Tran-Van-Minh, A., Ye, Z., &amp; Rancz, E. (2023). Quantitative analysis of rabies virus-based synaptic connectivity tracing. <em>PLoS One</em>, <em>18</em>(3), e0278053.</p>
<p>Tsoi, S. Y., Öncül, M., Svahn, E., Robertson, M., Bogdanowicz, Z., McClure, C., &amp; Sürmeli, G. (2021). Telencephalic outputs from the medial entorhinal cortex are copied directly to the hippocampus. <em>BioRxiv</em>, 2021–03.</p>
<p>Tsoi, S. Y., Öncül, M., Svahn, E., Robertson, M., Bogdanowicz, Z., McClure, C., &amp; Sürmeli, G. (2022). Telencephalic outputs from the medial entorhinal cortex are copied directly to the hippocampus. <em>Elife</em>, <em>11</em>, e73162.</p>
<p>Tyson, A. L., &amp; Margrie, T. W. (2022). Mesoscale microscopy and image analysis tools for understanding the brain. <em>Progress in Biophysics and Molecular Biology</em>, <em>168</em>, 81–93.</p>
<p>Tyson, A. L., Rousseau, C. V., Niedworok, C. J., Keshavarzi, S., Tsitoura, C., Cossell, L., Strom, M., &amp; Margrie, T. W. (2021). A deep learning algorithm for 3D cell detection in whole mouse brain image datasets. <em>PLoS Computational Biology</em>, <em>17</em>(5), e1009074.</p>
<p>Tyson, A. L., Vélez-Fort, M., Rousseau, C. V., Cossell, L., Tsitoura, C., Lenzi, S. C., Obenhaus, H. A., Claudi, F., Branco, T., &amp; Margrie, T. W. (2022). Accurate determination of marker location within whole-brain microscopy images. <em>Scientific Reports</em>, <em>12</em>(1), 867.</p>
<p>Tyson, A. L., Vélez-Fort, M., Rousseau, C. V., Cossell, L., Tsitoura, C., Obenhaus, H. A., Claudi, F., Lenzi, S. C., Branco, T., &amp; Margrie, T. W. (2021). Tools for accurate post hoc determination of marker location within whole-brain microscopy images. <em>BioRxiv</em>, 2021–05.</p>
<p>Vohra, S. K., Harth, P., Isoe, Y., Bahl, A., Fotowat, H., Engert, F., Hege, H.-C., &amp; Baum, D. (2023). A visual interface for exploring hypotheses about neural circuits. <em>IEEE Transactions on Visualization and Computer Graphics</em>.</p>
<p>Wagner, R., &amp; Rohr, K. (2022). Cellcentroidformer: Combining self-attention and convolution for cell detection. <em>Annual Conference on Medical Image Understanding and Analysis</em>, 212–222.</p>
<p>Walker, L. A., Williams, J. S., Li, Y., Roossien, D. H., Lee, W. J., Michki, N. S., &amp; Cai, D. (2022). ngauge: Integrated and extensible neuron morphology analysis in python. <em>Neuroinformatics</em>, <em>20</em>(3), 755–764.</p>
<p>Wang, L.-W., Wu, Y.-L., Lee, C.-L., Cheng, C.-C., Lu, K.-Y., Tsai, J.-H., Lin, Y.-H., Hsu, C.-H., Kuo, T.-H., &amp; Chu, L.-A. (2023). A Weakly Supervised U-Net Model for Precise Whole Brain Immunolabeled Cell Detection. <em>BioRxiv</em>, 2023–03.</p>
<p>Wang, Z., Romanski, A., Mehra, V., Wang, Y., Brannigan, M., Campbell, B. C., Petsko, G. A., Tsoulfas, P., &amp; Blackmore, M. G. (2022). Brain-wide analysis of the supraspinal connectome reveals anatomical correlates to functional recovery after spinal injury. <em>Elife</em>, <em>11</em>, e76254.</p>
<p>Wang, Z., Romanski, A., Mehra, V., Wang, Y., Campbell, B. C., Petsko, G. A., Tsoulfas, P., &amp; Blackmore, M. (2021). Brain-wide quantification of the supraspinal connectome. <em>BioRxiv</em>, 2021–06.</p>
<p>Weiler, S., Rahmati, V., Isstas, M., Wutke, J., Stark, A. W., Franke, C., Graf, J., Geis, C., Witte, O. W., Hübener, M., &amp; others. (2024). A primary sensory cortical interareal feedforward inhibitory circuit for tacto-visual integration. <em>Nature Communications</em>, <em>15</em>(1), 3081.</p>
<p>Weiler, S., Teichert, M., &amp; Margrie, T. W. (2024). Layer 6 corticocortical cells dominate the anatomical organization of intra and interhemispheric feedback. <em>BioRxiv</em>, 2024–05.</p>
<p>Wilde, M., Constantin, L., Thorne, P. R., Montgomery, J. M., Scott, E. K., &amp; Cheyne, J. E. (2022). Auditory processing in rodent models of autism: a systematic review. <em>Journal of Neurodevelopmental Disorders</em>, <em>14</em>(1), 48.</p>
<p>Xie, Z., Wang, M., Liu, Z., Shang, C., Zhang, C., Sun, L., Gu, H., Ran, G., Pei, Q., Ma, Q., &amp; others. (2021). Transcriptomic encoding of sensorimotor transformation in the midbrain. <em>Elife</em>, <em>10</em>, e69825.</p>
<p>Yang, W., Kanodia, H., &amp; Arber, S. (2023). Structural and functional map for forelimb movement phases between cortex and medulla. <em>Cell</em>, <em>186</em>(1), 162–177.</p>
<p>Zahler, S. H., Taylor, D. E., Wright, B. S., Wong, J. Y., Shvareva, V. A., Park, Y. A., &amp; Feinberg, E. H. (2023). Hindbrain modules differentially transform activity of single collicular neurons to coordinate movements. <em>Cell</em>, <em>186</em>(14), 3062–3078.</p>
<p>Zhang, Y., Ackels, T., Pacureanu, A., Zdora, M.-C., Bonnin, A., Schaefer, A. T., &amp; Bosch, C. (2022). Sample preparation and warping accuracy for correlative multimodal imaging in the mouse olfactory bulb using 2-photon, synchrotron x-ray and volume electron microscopy. <em>Frontiers in Cell and Developmental Biology</em>, <em>10</em>, 880696.</p>
<p>Zhou, H., Li, Y., Wen, W., Yang, H., Ma, Y., &amp; Chen, M. (2024). Accurately 3D neuron localization using 2D conv-LSTM super-resolution segmentation network. <em>IET Image Processing</em>, <em>18</em>(2), 535–547.</p>
<p>Zhu, Y., Wang, G., Kolluru, C., Gu, Y., Gao, H., Zhang, J., Wang, Y., Wilson, D. L., Zhu, X., Flask, C. A., &amp; others. (n.d.). <em>This manuscript was submitted to JCBFM for review on Nov 3, 2022.</em></p>
<p>Zhu, Y., Wang, G., Kolluru, C., Gu, Y., Gao, H., Zhang, J., Wang, Y., Wilson, D. L., Zhu, X., Flask, C. A., &amp; others. (2023). Transport pathways and kinetics of cerebrospinal fluid tracers in mouse brain observed by dynamic contrast-enhanced MRI. <em>Scientific Reports</em>, <em>13</em>(1), 13882.</p>
<p>Zhukovskaya, A., Zimmerman, C. A., Willmore, L., Janarthanan, S. N., Falkner, A. L., &amp; Witten, I. B. (2023). Differences in an aversive teaching signal produce brainwide and behavioral substrates of susceptibility. <em>BioRxiv</em>, 2023–11.</p>
<p>Zimmerman, C. A., Pan-Vazquez, A., Wu, B., Keppler, E. F., Guthman, E. M., Fetcho, R. N., Bolkan, S. S., McMannon, B., Lee, J., Hoag, A. T., &amp; others. (2023). A neural mechanism for learning from delayed postingestive feedback. <em>BioRxiv</em>.</p>
</section>

<div class="section ablog__blog_comments">
   
</div>

                </article>
              
              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="contact.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Contact</p>
      </div>
    </a>
    <a class="right-next"
       href="talks.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Talks</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <dialog id="pst-secondary-sidebar-modal"></dialog>
                <div id="pst-secondary-sidebar" class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
<div
    id="pst-page-navigation-heading-2"
    class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> On this page
  </div>
  <nav class="bd-toc-nav page-toc" aria-labelledby="pst-page-navigation-heading-2">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#">BrainGlobe publications</a></li>
<li class="toc-h1 nav-item toc-entry"><a class="reference internal nav-link" href="#publications-citing-brainglobe">Publications citing BrainGlobe</a></li>
</ul>

  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
          </footer>
        
      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script defer src="_static/scripts/bootstrap.js?digest=8878045cc6db502f8baf"></script>
<script defer src="_static/scripts/pydata-sphinx-theme.js?digest=8878045cc6db502f8baf"></script>

  <footer class="bd-footer">
<div class="bd-footer__inner bd-page-width">
  
    <div class="footer-items__start">
      
        <div class="footer-item"><p class="sphinx-version">
Created using <a href="https://sphinx-doc.org/">Sphinx</a> 8.1.3.
<br/>
</p>
<p class="theme-version">
Built with the
<a href="https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html">PyData Sphinx Theme</a>
0.16.1.
</p></div>
      
    </div>
  
  
  
    <div class="footer-items__end">
      
        <div class="footer-item"><div class="things-in-a-row">
</div></div>
      
    </div>
  
</div>

  </footer>
  </body>
</html>